endogenous/Snakefile

```diff
@@ -0,0 +1,11 @@
+configfile: "config.yaml"
+
+wildcard_constraints:
+    simpl="[a-zA-Z0-9]*",
```

Comment:
Unless overlooking something, wildcard/regular expression _{simpl}_ could be removed since it is not needed  (i.e. the CSV file being consumed, _transport_demand_s_45.csv_, does not have a number right after the letter _s_). Most likely, the inclusion of this wildcard/regular expression was done to follow PyPSA-Eur-Sec own _config.yaml_ file content.

On a side note, when looking at the name of the CSV files PyPSA-Eur-Sec consume (transport-wise) it seems that there is no number right after the letter _s_. Maybe this could be refactored in PyPSA-Eur-Sec as well.